File: Assuit-Sheet/First-Contest/H.cpp
#include <iostream>

using namespace std;

int main()
{
   double a , k , n ;
   cin >> n >> k >> a ;
   long long X = (k * n) / a ;
   double q = (k*n)/a , test = q - X ;
   /* if(test > 0)
   {
      cout <<"double"<< endl ;
   }
   else      if (q > 2147483647)
      cout <<"long long" << endl ;
   else
      cout<<"int"<<endl; */

   cout << X << endl ;
   cout << q << endl ;
   cout << test << endl ;
   
   return 0;
}

File: Assuit-Sheet/First-Contest/H.py
n, k, a = map(int, input().split())

X = int((k*n)/a) 
q = float((k*n)/a)
test = q - X

if test > 0:
    print("double")
elif q > 2147483647:
    print("long long")
else:
    print("int")
